hooks/pre_gen_project.py

```diff
@@ -0,0 +1,12 @@
+"""
+    This script has as purpose to create the Python package name and the 
+    'Inmanta module name' using the module name inputed by the user (or the default module name).
```

Comment:
```suggestion
    'Inmanta module name' using the module name inputted by the user (or the default module name).
```

Reply:
I believe plain `input` is what you're looking for.